### PR TITLE
MULE-18111: Add information about the last commit included in the distro build

### PR DESCRIPTION
**Pull request by JuLuzzi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,6 @@ testdir/
 hostkey.ser
 *.h2.db
 *.factorypath
+
+# Manifest
+src/main/resources/META-INF/MANIFEST.MF
```
Comment (anosenzo) [inline .gitignore:35]: Why did you need to include this line in the .gitignore?

Comment (elrodro83) [inline .gitignore:35]: is this needed?
